source/textUtils.py

```diff
@@ -43,31 +130,20 @@ class WideStringOffsetConverter:
 	_bytesPerIndex: int = ctypes.sizeof(ctypes.c_wchar)
 
 	def __init__(self, text: str):
-		super().__init__()
-		if not isinstance(text, str):
-			raise TypeError("Value must be of type str")
-		self.decoded: str = text
+		super().__init__(text)
 		self.encoded: bytes = text.encode(self._encoding, errors="surrogatepass")
 
-	def __repr__(self):
-		return "{}({})".format(self.__class__.__name__, repr(self.decoded))
-
 	@property
-	def wideStringLength(self) -> int:

```

Comment:
Removal of this property will be backward compatibility breaking. Some add-ons may rely on it (certainly one of mine does). Could you implement a back compat getter which logs the information that `wideStringLength` is deprecated and returns result of the new property? This should also be documented in the compatibility section in the change log after this is no longer a draft.

Reply:
Added back old function names as aliases of new ones to preserve compatibility.